lib/derive-idol-err/Cargo.toml

```
@@ -6,6 +6,7 @@ edition = "2021"
 [dependencies]
 syn = { workspace = true }
 quote = { workspace = true }
+proc-macro2 = {workspace = true}
```

Nit: to match the other lines,
```suggestion
proc-macro2 = { workspace = true }
```

---

Is there a tool to maintain this? I've made the change in this commit, but I'm not excited about maintaining non-semantic whitespace through code review going forward.

---

I did some searching for an automatic TOML formatter a while ago, and came up empty.  It looks like IntelliJ has [something](https://github.com/intellij-rust/intellij-rust/pull/5536), but not sure whether it's usable as a standalone tool...